raft-tech/TANF-app
# Knowledge center img fix

## Summary of Changes
Corrects corrupted images in Knowledge Center. 
Commits are a bit messy since they reflect the whole process of troubleshooting the issue (which didn't exist in my test environment). The issue was gitattributes in the root level of the repo treating jpg as binary but not jpeg. I went ahead and changed all relevant images from jpg to jpeg since they're functionally identical from a visual standpoint—I believe it would also work to add support for jpegs to the treat as binary section of our gitattributes file but I wasn't sure as to whether that had broader implications. 

## How to Test
Most of the affected images were located in this folder: https://github.com/raft-tech/TANF-app/tree/knowledge-center-img-fix/product-updates/img/complete-data

Testing:
1. Confirm that jpegs have been changed to jpgs
2. Open image file in Github to preview 

The PR changed file comparison view also shows a nice before and after.
